internal/common/common_test.go

```diff
@@ -160,3 +163,44 @@ func TestGetSysctrlEnv(t *testing.T) {
 		t.Errorf("unexpected real result from getSysctrlEnv: %q", env)
 	}
 }
+
+func TestGetEnvDefault(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("windows doesn't have etc")
+	}
+	p := HostEtcWithContext(context.Background(), "mtab")
+	if p != "/etc/mtab" {
+		t.Errorf("invalid HostEtc, %s", p)
+	}
+}
+
+func TestGetEnvWithNoContext(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("windows doesn't have etc")
+	}
+	old := os.Getenv("HOST_ETC")
+	os.Setenv("HOST_ETC", "/bar")
+	defer func() {
+		os.Setenv("HOST_ETC", old)
+	}()
```

Comment:
How about to use [testing.Setenv](https://pkg.go.dev/testing#T.Setenv) instead?

Reply:
Done.